rental.rb

```diff
@@ -1,14 +1,11 @@
 class Rental
-  attr_accessor :person, :date
-  attr_reader :book
+  attr_accessor :person, :date, :book
 
   def initialize(date, person, book)
     @date = date
-
     @person = person
-    person.rentals << self
-
     @book = book
+    person.rentals << self
     book.rentals << self
   end
```

Comment:
- [ ] When I try to create a rental, I get an error. Kindly check it out

![image](https://user-images.githubusercontent.com/98453979/224477214-3d36d970-d73e-4975-b8e7-486d08e74146.png)
![image](https://user-images.githubusercontent.com/98453979/224477259-2bc33b53-c8d8-419d-a831-89ead36367e5.png)